rust_src/src/alloc.rs

```diff
@@ -112,4 +114,21 @@ pub fn purecopy(obj: LispObject) -> LispObject {
     }
 }
 
+/// Return a newly allocated marker which does not point to any place.
+#[lisp_fn]
+pub fn make_marker() -> LispMarkerRef {
+    //let val: LispObject = unsafe { allocate_misc(Lisp_Misc_Type::Lisp_Misc_Marker) };
+    //let mut p: LispMarkerRef = val.as_marker().unwrap();
+    let mut p = unsafe { allocate_misc(Lisp_Misc_Type::Lisp_Misc_Marker) }.force_marker();
```

Comment:
This is a slightly pedantic point, but it would be nice if you could give `p` a slightly more semantically meaningful name like `mark` or `marker`.